packages/filebrowser/src/crumbs.ts

```diff
@@ -370,9 +370,9 @@ namespace Private {
    */
   export function createCrumbSeparators(): ReadonlyArray<HTMLElement> {
     let items: HTMLElement[] = [];
-    for (let i = 0; i < 3; i++) {
-      let item = document.createElement('i');
-      item.className = 'fa fa-angle-right';
+    for (let i = 0; i < 4; i++) {
```

Comment:
This magic number of four threw me -- perhaps we can have a `MAX_DIRECTORIES` const or something (to be used above), and have this be `MAX_DIRECTORIES+2`?

Reply:
It also threw me. I was trying to not make too many changes - not rock the boat too much at this stage of the game.

I think the crumbs and filebrowser needs to be rewritten anyway.

Reply:
I just added some comments and the const like you asked.

Reply:
A nit, but `MAX_DIRECTORIES` could also be used above in assembling the crumbs.

Reply:
Not without rewriting the code above. The code structure itself hardcodes the notion that only two directories are being shown.

Again, I was trying to not rewrite everything at this stage :).

Reply:
(i.e., the loop you might expect above is unrolled as a handcoded if statement)

Reply:
Fair enough. It's a pretty confusing construction as it stands, but we shouldn't block on it.